Add separate use cases section (#120)
decentralized-identity/credential-manifest: spec/spec.md

```
:::

## Use Cases

### Credential Issuance Discovery

> As an Issuer, I want to provide a list of credential I offer, which wallets and other entities can find and digest.

Reviewer: ```suggestion
As an Issuer, I want to provide a list of credentials I offer, for wallets and other entities to find and digest.
```

### Credential Requirement Discovery

> As an Issuer, I want to provide a list of VC requirements that a Holder would need to submit to acquire a given credential.

Reviewer: ```suggestion
As an Issuer, I want to provide a description of the information that a Holder would need to submit to acquire a given credential.
```
